alerter/ThresholdAlerterStub.java

```diff
@@ -0,0 +1,6 @@
+
+
+public interface ThresholdAlerterStub {
```

Comment:
Primary reason for defining interface is to abstract implementation. In this usecase, the method could be a test stub or production code. Hence defining an interface is the right approach 👍 

But naming should not contain stub, the reason is that the object deriving from interface could be either test object or production object.
Please rename ThresholdAlerterStub suitably.